en, zh: update docs for canary upgrade (#1345)
pingcap/docs-tidb-operator: en/deploy-multiple-tidb-operator.md

> **Note:**
>

> > - Currently, you can only deploy multiple `tidb-controller-manager`s and `tidb-scheduler`s, AdvancedStatefulSet controller and AdmissionWebhook are not supported.

Reviewer: ```suggestion
> - Currently, you can only deploy multiple sets of `tidb-controller-manager` and `tidb-scheduler`. Deploying multiple sets of AdvancedStatefulSet controller and `tidb-admission-webhook` is not supported.
```

    appendReleaseSuffix: true
    scheduler:
      create: false
    advancedStatefulset:
      create: false
    admissionWebhook:
      create: false
    ```

    > **Note:**
    >
    > * It is recommended to deploy the new TiDB Operator in a separate namespace.
    > * Set `appendReleaseSuffix` to `true`.
    > * If you configure `scheduler.create: true`, a `tidb-scheduler` named `{{ .scheduler.schedulerName }}-{{.Release.Name}}` is created. To use this `tidb-scheduler`, you need to configure `spec.schedulerName` in the `TidbCluster` CR to the name of this scheduler.

>     > * You need to set `advancedStatefulset.create: false` and `admissionWebhook.create: false` because you cannot deploy multiple sets of AdvancedStatefulSet controller and AdmissionWebhook.

Reviewer: ```suggestion
    > * You need to set `advancedStatefulset.create: false` and `admissionWebhook.create: false`, because deploying multiple sets of AdvancedStatefulSet controller and `tidb-admission-webhook` is not supported.
```
